### PR TITLE
Fix the issue: Image on post not full when using mobile-view and make rich content support better in mobile-view 

### PR DESCRIPTION
**Pull request by RuifMaxx**
Fix the issue: Image on post not full when using mobile-view and make rich content support better in mobile-view 
H@R%~E4W_CG8BA8I KK}3CD
PCL{ G`3Y46@E`GR__`_Q1P

### CODE REVIEW

**Comment by RuifMaxx** [comment]
max- width:100% and width:100% The difference is that max width is relative to the size of img itself. It means that the maximum width of a picture is the width of its own size.  We only allow the IMG smaller, but not to be enlarged.

File: exampleSite/content/post/post.1.en.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
...
+description = 'Example of post with missing date property butwith pic and video.'
 disableComments = true
 +++
-Example of post with missing date property.
\ No newline at end of file
+Example of post with missing date property but with pic and video.
+
+![img](/images/pic03.jpg)
+
+<div class="meta-media"><iframe src="//player.bilibili.com/player.html?aid=373841893&bvid=BV1Wo4y1R76n&cid=287681080&page=1" frameborder="no" scrolling="yes" allowfullscreen="allowfullscreen" high_quality="1" framespacing="1" class="video" > </iframe></div>
```
Comment (curtiscde) [inline exampleSite/content/post/post.1.en.md:12]: Having to add HTML into the markdown files is not ideal. Fortunately I think Hugo provides a way of handling this with shortcodes:

https://gohugo.io/content-management/shortcodes

Comment (curtiscde) [reply exampleSite/content/post/post.1.en.md:12]: I've also discovered a bug with this when removing the content beneath the video:

image

**Review by curtiscde — APPROVED** [review]
For now, I have removed the references to the video files, and just included the fix for images.

I've also moved this to a new file so that it won't override the change in the future if the `reset.scss` file is updated to a newer version.

Thanks for the contribution! 🎉

**Pull request merged**